00E028.py

```diff
@@ -15,6 +15,7 @@
 # speeds in miles per hour.                                   *
```

Comment:
Status should now be in testing.

Reply:
Will update the status in the next iteration. Program needs to be tested first.